publish.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -euo pipefail
+
+LABEL_PUBLISH="publish-sdk"
+PACKAGE_GLOB=(./packages/lib-*)
+NPM_REGISTRY="https://registry.npmjs.org/"
+NPM_REGISTRY_AUTH_KEY="//registry.npmjs.org/:_authToken"
+LABELS=$(curl -s \
+  -H "Accept: application/vnd.github.v3+json" \
+  "https://api.github.com/repos/${REPO_OWNER}/${REPO_NAME}/pulls/${PULL_NUMBER}" \
+  | jq '.labels | map(.name)')
+HAS_PUBLISH=$(printf '%s\n' "${LABELS}" | jq 'index("'${LABEL_PUBLISH}'")')
+
+if [[ "${HAS_PUBLISH}" != "null" ]]; then
+  echo 'publish NPM packages'
+  if [[ "${JOB_TYPE}" != "postsubmit" ]]; then
```

Comment:
Do we even want to run the publish on non-postsubmit jobs? The dry run could be usefull, but I am a bit worried about the length of this task to run.

Reply:
@karelhala since this is gated by the logic above checking if the PR has the label `publish-sdk`, it will only run for the pre and post submits for PRs that include the `publish-sdk` label. Or am I missing something?